doc-content/enterprise-only/installation/jms-broker-configure_proc.adoc

```diff
@@ -0,0 +1,25 @@
+[id='JMS-broker-configure_{context}']
+= Downloading and configuring AMQ Broker
+
+AMQ Broker enables your applications to communicate with any messaging provider. It specifies how components such as message-driven beans, Enterprise JavaBeans, and servlets can send or receive messages. 
+
+.Procedure
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+* *Product: AMQ BROKER*
+* *Version: 7.2*
+. Click *Download* next to *Red Hat AMQ Broker 7.2.1* (`amq-broker-7.2.1-bin.zip`).
+. Extract the `amq-broker-7.2.1-bin.zip` file. 
+. Change directory to `amq-broker-7.2.0-bin/amq-broker-7.2.1/bin`.
+. Enter the following command to create the AMQ Broker user, where `<username>` and `<password>` is a user name and password combination of your choice. COMMENT: What is `mybroker`?
```

Comment:
regarding COMMENT:
broker is a software component that receive messages, store them locally and forwards the message to a recipien

see [amq terminology page](http://activemq.apache.org/terminology.html)

in this case mybroker is the name of the broker.
Depending on the name you choose the command to run the broker will differ (see on line 23 in this doc).
e.g. if you create broker 

`create --host <ip-address-or-hostname> --user <username>` --password <password> --require-login <broker-name>`

You'll have to start it with
`amq-broker-7.2.1/<broker-name>/bin/artemis run`